src/utils/constants.ts

```diff
@@ -284,16 +282,17 @@ export default {
   LOOKUP_TABLE_HEADER_ACTIONS: 'Actions',
   LOOKUP_TABLE_VIEW: 'View lookup table',
   LOOKUP_TABLE_IMPORT: 'Import lookup table',
-  LOOKUP_TABLE_IMPORT_INTO: 'Import into Lookup-table: %1',
+  LOOKUP_TABLE_IMPORT_INTO: 'Import into Lookup-table',
   LOOKUP_TABLE_IMPORT_OTHER: 'Import another CSV',
   LOOKUP_TABLE_DOWNLOAD: "Download '%1' table",
   LOOKUP_TABLE_FIELD_NAME: 'Database Field-name',
   LOOKUP_TABLE_LABEL: 'Lookup-table Label',
   LOOKUP_TABLE_NAME: 'Table name',
   LOOKUP_TABLE_ADD: 'Add new Lookup-table',
-  LOOKUP_TABLE_TITLE: 'Lookup Table: %1',
+  LOOKUP_TABLE_TITLE: 'Lookup table',
   LOOKUP_TABLE_CONTENTS: 'View individual lookup-table and its contents',
   LOOKUP_TABLE_SUBTITLE: 'This page contains the list of all lookup-tables',
-  LOOKUP_TABLE_SUCCESS: "Lookup table '%1' has been successfully %2.",
+  LOOKUP_TABLE_SUCCESS_CREATED: 'has been successfully created.',
+  LOOKUP_TABLE_SUCCESS_UPDATED: 'has been successfully updated.',
```

Comment:
Hmm, I might be slightly contradicting myself here, but I think "full sentences" should take priority over minimising substitutions. 

So perhaps these two strings should be:
- `Lookup Table %1  has been successfully created.`
- `Lookup Table %1  has been successfully updated.`

I think it's easier to read and simpler for a translator to follow, even with the %substitution in it. What do you think?

Reply:
Not a big difference to me. So ok to do as you suggested.

Reply:
Done 👍